### PR TITLE
update to latest api.proto

### PR DESCRIPTION
**Pull request by richardhopton**

### CODE REVIEW

File: aioesphomeapi/api.proto [hunk 1]
```diff
@@ -283,7 +284,6 @@ message ListEntitiesCoverResponse {
   bool disabled_by_default = 9;
   string icon = 10;
   EntityCategory entity_category = 11;
-  bool supports_stop = 12;
```
Comment (richardhopton) [inline aioesphomeapi/api.proto:286]: Perhaps this should be kept with `[deprecated = true]` or something

File: aioesphomeapi/api.proto [hunk 2]
```diff
@@ -610,7 +623,7 @@ message SubscribeLogsResponse {
   option (no_delay) = false;
 
   LogLevel level = 1;
-  bytes message = 3;
+  string message = 3;
```
Comment (richardhopton) [inline aioesphomeapi/api.proto:626]: This is a breaking change - but super convenient having the name coerced to a string.

File: aioesphomeapi/api.proto [hunk 3]
```diff
@@ -1166,12 +1183,13 @@ message BluetoothLEAdvertisementResponse {
   option (no_delay) = true;
 
   uint64 address = 1;
-  bytes name = 2;
+  string name = 2;
```
Comment (richardhopton) [inline aioesphomeapi/api.proto:1186]: This is another breaking change - but super convenient having the name coerced to a string.

**Comment by richardhopton** [comment]
I didn't remove the siren stuff that @jesserockz added to this repo but not to esphome repo.

File: aioesphomeapi/api.proto (hunk 1 above)
Comment (jesserockz) [reply aioesphomeapi/api.proto:286]: Why are you removing this field? It was only just added recently

File: aioesphomeapi/api.proto (hunk 3 above)
Comment (jesserockz) [reply aioesphomeapi/api.proto:1186]: https://github.com/esphome/aioesphomeapi/pull/339

File: aioesphomeapi/api.proto (hunk 1 above)
Comment (richardhopton) [reply aioesphomeapi/api.proto:286]: I copied the api.proto from here: https://github.com/esphome/esphome/blob/dev/esphome/components/api/api.proto
and these were all the things that changed... I was clearly incorrect to think that keeping these two files in sync was a good idea.

**Pull request closed without merging**